README.md

```diff
@@ -41,26 +41,38 @@
        virtual environment. This is done by using the command 
        (Set-ExecutionPolicy -ExecutionPolicy RemoteSigned)
 
-    1. Run setup via command line, in the form (python setup.py (Mac/Linux)) (py setup.py (Windows))
+    1. Creating and entering a virtual environment
 
-    2. setup.py will create a viurutal environment, instatiate databases,
-       and intall any required non-base modules/packages into the environment
+        Activating
+            Linux/Mac
+                -use the command line command (python3 -m venv spell)
+            Windows
+                -Use the command line command (py -m venv spell)
```

Comment:
Parens '()' could be confusing, maybe use ' ?

Reply:
Changed style to better emphasize commands